src/components/Card.vue

```diff
@@ -0,0 +1,317 @@
+    eyebrowId: ({ _uid }) => `card_eyebrow_${_uid}`,
+    linkAriaTags: ({ titleId, eyebrowId, contentId }) => ({
+      'aria-labelledby': `${titleId} ${eyebrowId}`,
+      'aria-describedby': `${contentId}`,
+    }),
+    classes: ({
+      size,
+      floatingStyle,
+    }) => ([
+      size,
+      {
+        'floating-style': floatingStyle,
+      },
+    ]),
+    imageReference: ({
+      image,
+      references,
+    }) => (references[image] || {}),
+    imageVariants: ({ imageReference }) => imageReference.variants || [],
+  },
+  props: {
+    linkText: {
+      type: String,
+      required: false,
+    },
+    url: {
+      type: String,
+      required: false,
+      default: '',
+    },
+    eyebrow: {
+      type: String,
+      required: false,
+    },
+    image: {
+      type: String,
+      required: false,
+    },
+    size: {
+      type: String,
+      validator: s => Object.prototype.hasOwnProperty.call(CardSize, s),
+    },
+    title: {
+      type: String,
+      required: true,
+    },
+    hasButton: {
+      type: Boolean,
+      default: () => false,
+    },
+    floatingStyle: {
+      type: Boolean,
+      default: false,
+    },
+    showExternalLinks: {
+      type: Boolean,
+      default: false,
+    },
+    formatAriaLabel: {
+      type: Function,
+      default: v => v,
+    },
+  },
+};
+</script>
+
+<style scoped lang="scss">
+@import 'docc-render/styles/_core.scss';
+
+$details-padding: 17px;
+$content-margin: 4px;
+
+@mixin static-card-size($card-height, $img-height) {
+  @include inTargetWeb {
+    --card-height: #{$card-height};
+    --card-details-height: #{$card-height - $img-height - ($details-padding * 2)};
+  }
+  --card-cover-height: #{$img-height};
+}
+
+.card {
+  overflow: hidden;
+  display: block;
+  transition: box-shadow, transform 160ms ease-out;
+  will-change: box-shadow, transform;
+  backface-visibility: hidden;
+  height: var(--card-height);
+
+  &:hover {
+    text-decoration: none;
+
+    .link {
+      text-decoration: underline;
+    }
+  }
+
+  @include inTargetWeb {
+    border-radius: $big-border-radius;
+
+    &:hover {
+      box-shadow: 0 5px 10px var(--color-card-shadow);
+      transform: scale(1.007);
+
+      @media (prefers-reduced-motion: reduce) {
+        box-shadow: none;
+        transform: none;
+      }
+    }
+  }
+
+  &.small {
+    @include static-card-size(408px, 235px);
+    @include breakpoint(medium) {
+      @include static-card-size(341px, 163px);
+    }
+  }
+
+  &.large {
+    @include static-card-size(556px, 359px);
+    @include breakpoint(medium) {
+      @include static-card-size(420px, 249px);
+    }
+  }
+
+  &.floating-style {
+    --color-card-shadow: transparent;
+    --card-height: auto;
+    --card-details-height: auto;
+  }
+}
+
+.details {
+  background-color: var(--color-card-background);
+  padding: $details-padding;
+  position: relative;
+  height: var(--card-details-height);
+
+  .small &, & {
```

Comment:
What do you mean by adding the second `&` to it?

Reply:
Transpiles to:
```css
.details, .small .details {

}
```
It is basically a default state

Reply:
if it's the default state, why do you need the `small` class at all since it's not overwriting it?